src/browser/modules/Stream/CypherFrame/RelatableView/relatable-view.tsx

```diff
@@ -90,12 +91,12 @@ function getColumns(records: Record[], maxFieldItems: number) {
 
   return map(keys, key => ({
     Header: key,
-    accessor: (record: Record) => {
+    accessor: (record: Record): string => {
       const fieldItem = record.get(key)
 
       if (!Array.isArray(fieldItem)) return fieldItem
 
-      return slice(fieldItem, 0, maxFieldItems)
+      return slice(fieldItem, 0, maxFieldItems).join(',')
```

Comment:
I remember this is something we tried to get the types to work properly. But we should remove it, as it causes the following bug:
<img width="896" alt="CleanShot 2022-02-28 at 12 12 09@2x" src="https://user-images.githubusercontent.com/10564538/155973666-e34d0094-6a89-412e-a225-5007bb4a3e46.png">


How it should look: 
<img width="888" alt="bild" src="https://user-images.githubusercontent.com/10564538/155973615-e8a15ed1-da3e-43f0-b5f1-6ebce521f880.png">

Reply:
(I tested and it works fine when we change it back, the :string type annotation also needs to be removed)

Reply:
ah yes, good catch, thanks!